sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryLevelExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package io.sentry.kotlin.multiplatform.extensions
+
+import io.sentry.kotlin.multiplatform.AndroidSentryLevel
+import io.sentry.kotlin.multiplatform.SentryLevel
+
+fun SentryLevel.toAndroidSentryLevel(): AndroidSentryLevel {
```

Comment:
I'd rather move those extensions inside of `SentryLevel`

Reply:
There is no SentryLevel on Android or iOS modules (it's only an enum) so I can't move them elsewhere